std/src/main/scala/algebra/std/int.scala

```diff
@@ -8,6 +9,11 @@ package object int extends IntInstances
 
 trait IntInstances {
   implicit val intAlgebra = new IntAlgebra
+
+  val IntMinMaxLattice = new Lattice[Int] {
```

Comment:
stylewise, this looks weird. While I personally often do this, we seem to have a named class style so far. Curious why this one is anonymous.